lane_planner:
    map_frame_id : map
    lane_width : 0.7
    vehicle_width : 0.4
    sampling_number : 7
    distanceCost_gain : 1.0
    obstacleCost_gain : 10
    markerSize : 1.0
    maxVelocity : 0.7
    distanceTarget_maxVelocity : 1
    distance_rate : 0.4
    loop_rate : 20